app/api/mapping/services.py

```diff
@@ -46,3 +47,17 @@ def download_data_dictionary_blob(blob_name, container="data-dictionaries"):
     response["Content-Disposition"] = f'attachment; filename="{blob_name}"'
 
     return response
+
+
+def delete_blob(blob_name, container):
```

Comment:
Please add type annotations (both the arguments, and return type) and a docstring, we're setting a new standard for the codebase, and any code introduced should use them.